src/ring_theory/discrete_valuation_ring.lean

```diff
@@ -202,20 +200,23 @@ let spec := classical.some_spec hR in
     left,
     obtain ⟨m, rfl⟩ := nat.exists_eq_succ_of_ne_zero hm,
     apply dvd_mul_of_dvd_left (dvd_refl _) _,
```

Comment:
GitHub doesn't like suggestions spanning non-changed and changed lines, but the idea is to move the lonely brace on the next line one line up.
```suggestion
    apply dvd_mul_of_dvd_left (dvd_refl _) _ },
```